sdk/src/wallet/operations/transaction/high_level/minting/create_native_token.rs

```diff
@@ -170,7 +170,10 @@ where
                     )));
 
                     if let Some(foundry_metadata) = params.foundry_metadata {
-                        foundry_builder = foundry_builder.add_immutable_feature(MetadataFeature::new(foundry_metadata)?)
+                        foundry_builder = foundry_builder.add_immutable_feature(MetadataFeature::new(
+                            // TODO: what hardcoded key or let user provide the full metadata?
+                            [(b"foundry".to_vec(), foundry_metadata)],
```

Comment:
When I asked Max before the holidays he said `data`, but I didn't ask specifically for Foundry metadata, maybe ask him

Reply:
Nothing defined yet, I just changed it so the user can define it here